app/uk/gov/hmrc/customsexportsinternalfrontend/config/ErrorHandler.scala

```diff
@@ -22,10 +22,10 @@ import play.api.i18n.MessagesApi
 import play.api.mvc.Request
 import play.twirl.api.Html
 import uk.gov.hmrc.play.bootstrap.http.FrontendErrorHandler
-import uk.gov.hmrc.customsexportsinternalfrontend.views.html.error_template
+import uk.gov.hmrc.customsexportsinternalfrontend.views.html.error
```

Comment:
I think name `error_template` was more readable. I know it's lower case, you shouldn't make mistake with Error as class, but still `error_template` is more intuitive for me.

Reply:
I dont mind suffixing it with something but I feel that "template" is the wrong word